src/sinks/elasticsearch.rs

```diff
@@ -657,6 +659,10 @@ mod integration_tests {
     }
 
     fn run_insert_tests(mut config: ElasticSearchConfig, break_events: bool) {
+        // Need async version of ElasticSearchCommon::parse_config otherwise
+        // function called in blocked runtime will panic with error:
+        // default Tokio reactor already set for execution context
+
```

Comment:
I'm not sure, but probably elasticsearch on aws sink currently broken. If sink build called outside runtime block (`rt.block_on`) `ElasticSearchCommon::parse_config` works, but if we parse config happened in block vector will panic (see message in code).
It's not only about rusoto:0.44, I tried with 0.41 too. Test is ok because parse outside block_on.